app/src/main/java/com/stevesoltys/seedvault/settings/SettingsFragment.kt

```diff
@@ -105,7 +105,7 @@ class SettingsFragment : PreferenceFragmentCompat() {
                     dialog.dismiss()
                 }
                 .setNegativeButton(R.string.settings_backup_apk_dialog_disable) { dialog, _ ->
-                    apkBackup.isChecked = enable
+                    apkBackup.isChecked = false
```

Comment:
Why this change? It does make sense.

Reply:
The linter suggested that, because `enable` is always `false` here.